esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -197,6 +197,10 @@ EspHttpBinding::EspHttpBinding(IPropertyTree* tree, const char *bindname, const
                 {
                     //This is a Pluggable Security Manager
                     m_secmgr.setown(SecLoader::loadPluggableSecManager(bindname, bnd_cfg, secMgrCfg));
+                    if (!m_secmgr.get())
+                    {
+                        throw MakeStringException(-1, "Error loading Pluggable Security Manager : '%s'", m_authmethod.str());
```

Comment:
There are around 15 instances of "throw MakeStringException(-1," in this source file alone. I agree we should scrub the code to fix that but in this case I don't think so